pandas/tests/indexes/multi/test_reindex.py

```diff
@@ -104,3 +104,14 @@ def test_reindex_non_unique():
     msg = "cannot handle a non-unique multi-index!"
     with pytest.raises(ValueError, match=msg):
         a.reindex(new_idx)
+
+
+@pytest.mark.parametrize("values", [[["a"], ["x"]], [[], []]])
```

Comment:
this was user facing in a Series.reindex right? can you add the user facing part of this test as well.

Reply:
Done, but is essentially the same